dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -600,6 +600,14 @@ export const renderElement = ({
 				</Island>,
 			];
 		case 'model.dotcomrendering.pageElements.TweetBlockElement':
+			if (switches.enhanceTweets) {
+				return [
+					true,
+					<Island deferUntil="visible">
+						<TweetBlockComponent element={element} />
+					</Island>,
+				];
+			}
 			return [true, <TweetBlockComponent element={element} />];
```

Comment:
Wouldn’t that get two tweets?

Reply:
It's returning, so you can only get one. My linter insists on this pattern.